porting/finalize/UBports-official-support/port-support-requirements.rst

```diff
@@ -0,0 +1,102 @@
+.. _PortSupportRequirements:
+
+Port support requirements
+==================================
+
+This document enumerates the requirements that must be met in order for devices to be deemed ship ready for UBports releases.
+To be considered ready, port maintainers MUST meet the requirements presented in this document, including any documents incorporated via reference.
+
+The use of “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” is per the IETF standard defined in :RFC:`2119`.
+
+-----
+
+Attaining exceptions
```

Comment:
Pls move the Attaining exceptions section down so it is just before the current "Exception"  section. 

I think documents should always start to explain the 'normal' stuff first, so a new reader has an easier time grasping what we're talking about here.